netif.pyx

```diff
@@ -470,6 +470,18 @@ class CarpState(enum.IntEnum):
     MASTER = 2
 
 
+class BridgeMemberFlags(enum.IntEnum):
+    LEARN = defs.IFBIF_LEARNING
```

Comment:
Can we use LEARNING instead of LEARN to reflect that ifconfig output uses? Or was there a reason for this?

Reply:
I used `learn` because of the `ifconfig` command syntax. Should I shift it to `LEARNING` ?

Reply:
Actually we can keep consistent with the output. This is afterall a programming interface.